frontend/components/slot/viz/SlotVizViewer.vue

```diff
@@ -95,63 +101,122 @@ watch(props, () => {
 </script>
 <template>
   <div id="slot-viz" class="content">
-    <div class="rows">
-      <div class="row" />
-      <div v-for="row in props.data" :key="row.epoch" class="row">
-        <div class="epoch">
-          <BcFormatNumber :text="row.state === 'head' ? $t('slotViz.head') : formatNumber(row.epoch)" />
-        </div>
-      </div>
-    </div>
-    <div class="rows">
-      <div class="row">
+    <div class="header-row">
+      <BcTooltip class="info" :text="$t('slotViz.info_tootlip')" :dont-open-permanently="true">
+        <BcLink to="https://kb.beaconcha.in/v2beta/slot-visualization#how-does-it-work" target="_blank" class="link">
+          <FontAwesomeIcon :icon="faInfoCircle" />
+        </BcLink>
+      </BCTooltip>
+      <div class="filter-row">
         <BcToggleMultiBar v-model="selectedCategories" :icons="icons" />
```

Comment:
**Question:** The tooltips on mobile are tedious as they can only be viewed by pressing the button. But pressing the button also changes its toggle state so it is impossible to see the tooltip without performing an action.
We might want to address this issue with the product owner, right?

The same applies to the info icon to the left of these buttons of course.

Reply:
That's a general problem of tooltips on interactive items ... I mean we could prevent the action on the first click and only show the tooltip, but how does the user know he can click again? 

Tooltips are a bad UX for mobile and the UI should be intuitive enough without them and not rely on them. 

And that could be a general discussion and not for one specific instance ;-)